app/_hub/kong-inc/statsd/index.md

```diff
@@ -53,31 +60,35 @@ params:
   dbless_compatible: yes
   config:
     - name: host
-      required: false
+      required: true
       default: "`127.0.0.1`"
       value_in_examples: 127.0.0.1
+      datatype: string
       description: The IP address or host name to send data to.
     - name: port
-      required: false
+      required: true
       default: "`8125`"
       value_in_examples: 8125
-      description: The port to send data to on the upstream server
+      datatype: integer
+      description:  The port of StatsD server to send data to.
     - name: metrics
-      required: false
-      default: "All metrics<br>are logged"
+      required: true
+      default: "All metrics are logged"
+      datatype: Array of record elements
```

Comment:
I'm confused, the data type is an array, but the default value is a string?

Reply:
> I'm confused, the data type is an array, but the default value is a string?

@HCloward  This default has always been this way and is the same for other logging plugins such as statsd advanced, datadog, etc. It just means that by default, every single metric from the metrics sections below the param table is logged.